sdk/communication/azure-communication-email/azure/communication/email/_shared/utils.py

```diff
@@ -126,7 +127,7 @@ def get_authentication_policy(
         from azure.core.pipeline.policies import BearerTokenCredentialPolicy
         return BearerTokenCredentialPolicy(
             credential, "https://communication.azure.com//.default")
-    if isinstance(credential, str):
+    if isinstance(credential, AzureKeyCredential):
```

Comment:
Is this breaking to any existing support for `str` across other Communications SDKs?

Reply:
Good point, I'll make sure this works across SDKs.